test.js

```diff
@@ -0,0 +1,86 @@
+import {
+  addTask,
+  removeTask,
+  editTask,
+  updateTaskStatus,
+  clrCompletedTasks,
+} from './src/JSFile/addRmove.js';
+// Mock localStorage
+
+describe('Task manager', () => {
+  let tasks = [];
+  beforeEach(() => {
+    tasks = [
+      {
+        index: 0,
+        description: 'Task 1',
+        completed: false,
+      },
+      {
+        index: 1,
+        description: 'Task 2',
+        completed: false,
+      },
+      {
+        index: 2,
+        description: 'Task 3',
+        completed: false,
+      },
+    ];
+  });
+  test('addTask should add a new task', () => {
+    const task = {
+      index: 0,
+      description: 'This is Task 1',
+      completed: false,
+    };
+    tasks = addTask(task.description);
+    expect(tasks).toHaveLength(1);
+    expect(task.description).toBe('This is Task 1');
+    expect(task.index).toBe(0);
+    expect(task.completed).toBeFalsy();
+  });
+
+  test('removeTask should remove a task', () => {
+    const task = {
+      index: 1,
+      description: 'This is Task 2',
+      completed: false,
+    };
+    tasks = addTask(task.description);
+    tasks = removeTask(1, tasks);
+    tasks.forEach((task, index) => {
+      expect(task.index).toBe(index);
+    });
+    expect(tasks).toHaveLength(1);
+    expect(tasks).not.toContain(task);
+  });
+  test('editTask should edit a task', () => {
+    const task = {
+      index: 2,
+      description: 'This is Task 2',
+      completed: false,
+    };
+    tasks = addTask(task.description);
+    const str = 'This task is edited';
+    task.description = str;
+    tasks = editTask(2, str);
+    expect(task.description).toBe('This task is edited');
+  });
+  test('updateTaskStatus should update the status of a task', () => {
+    updateTaskStatus(0, true, tasks);
+    expect(tasks[0].completed).toBeTruthy();
+  });
+
+  test('clrCompletedTasks should clear completed tasks', () => {
+    const task = {
+      index: 2,
+      description: 'This is Task 2',
+      completed: true,
+    };
+    tasks = addTask(task.description);
+    clrCompletedTasks(tasks);
+    expect(tasks).toHaveLength(1);
+    expect(tasks).not.toContain(tasks[2]);
+  });
+});
```

Comment:
- [ ] Good job so far,  :clap: , However, as part of the project requirement, you are suppose to write separate files for each of the following tests: `editing the task` , `updating an item's 'completed' status` and `Clear all completed`. not combining them in one file and please ensure that each test file ends in `.test.js` . Please resolve it. :slightly_smiling_face: 

Kindly check the screenshot below. :slightly_smiling_face: 

![Sniip](https://github.com/Bupilipili/To-do-list/assets/43528594/a878770c-b8d0-4f98-998f-611a54fb4af1)